impl/client/src/test/javascript/pentaho/visual/base/view.spec.js

```diff
@@ -1011,6 +1011,24 @@ define([
 
         expect(view.isDirty).toBe(true);
       });
+
+      it("should become dirty on model change", function() {
+
+        view.__dirtyPropGroups.clear(); // view is clean
+
+        view.isAutoUpdate = false;
+
+        var didChangeHandler = jasmine.createSpy().and.callFake(function(){
+          expect(view.isDirty).toBe(true);
```

Comment:
Why is the `didChangeHandler ` needed? Couldn't this be tested bellow, after changing `model.selectionFilter`?

Reply:
The idea is making sure that the view knows it is dirty at that time — even before it's own did:change handler, that notifies it of the model having changed, is called.
Afterwards, it is surely dirty, and other tests already test(ed) that.

Reply:
👍